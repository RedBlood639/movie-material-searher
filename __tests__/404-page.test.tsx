import { renderApollo } from '@/lib/setupTests'
import Error404 from '@/pages/404'

describe('tv page', () => {
  it('should take a snapshot', () => {
    const { baseElement } = renderApollo(<Error404 />)

    expect(baseElement).toMatchSnapshot()
  })
})
