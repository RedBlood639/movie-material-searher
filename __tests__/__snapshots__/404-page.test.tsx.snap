// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tv page should take a snapshot 1`] = `
<body>
  <div>
    <div
      class="css-1dbltqw-Container exiooeo3"
    >
      <h1
        class="css-fqnkfq-Code exiooeo2"
      >
        404
      </h1>
      <div
        class="css-q8cjcr-TextWrapper exiooeo1"
      >
        <h2
          class="css-mm3euk-Text exiooeo0"
        >
          This page could not be found.
        </h2>
      </div>
    </div>
  </div>
</body>
`;
